bin/render-pdf.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * node.js script to render a given HTML file into the given PDF filename.
+ * Details of the API are at https://developers.google.com/web/tools/puppeteer/
+ *
+ * This command is invoked as:
+ *
+ *   $ node bin/render-pdf.js input.html output.pdf
+ *
+ */
+
+const puppeteer = require("puppeteer");
+
+const inputPath = `file://${process.argv[2]}`;
+const outputPath = process.argv[3];
+
+/* Travis needs an argument that is generally not prefered
+for live environments so we se it set it optionally here */
+const puppeteerArgs = process.env["CI"] ? ["--no-sandbox"] : [];
+
+console.log("Input path:", inputPath);
+console.log("Output path:", outputPath);
+
+(async () => {
+  const browser = await puppeteer.launch({args: puppeteerArgs});
+  const page = await browser.newPage();
+
+  console.log("Starting PDF conversion");
```

Comment:
is this meant to be left in prod?

Reply:
Yeah I think mostly just for logging purposes